protobufs/livekit_metrics.proto

```diff
@@ -46,6 +62,7 @@ message TimeSeriesMetric {
   uint32 participant_identity = 2; // index into `str_data`
   uint32 track_sid = 3; // index into `str_data`
   repeated MetricSample samples = 4;
+  uint32 rid = 5; // index into 'str_data'
```

Comment:
I think it is okay to do `string` for this if it makes it simple rather than making a `str_data` entry and indexing into it.

Reply:
Hmm, it's not a big deal either way, saves a bit on space as well.

Not familiar with protobufs, but does this need to be optional?

Reply:
Not really necessary I think. It won't get encoded on wire if it has default value.

Reply:
Since default value is zero, would that be an issue knowing whether rid is 0 or null?

Reply:
I guess it might just be better to do rid as string after all.

Reply:
The index starts at 4096 though, right?

Reply:
Ah right, forgot about that.